README.md

```diff
@@ -26,4 +26,52 @@ This will make building products among multiple cloud services and its services
 - Amazon EBS Backup Driver (Backup)
 ----
 
-## Intialize
+## Install
+
+> WIP
+
+## Usage
+
+```js
+const nodeCloud = require("nodecloud");
+// AWS
+const ncAWS = nodeCloud.getProvider("AWS");
+const options = {
+  apiVersion: "2016-11-15"
+};
+
+const params = {
+  ImageId: "ami-10fd7020", // amzn-ami-2011.09.1.x86_64-ebs
+  InstanceType: "t1.micro",
+  MinCount: 1,
+  MaxCount: 1
+};
+const instanceParams = {
+  Key: "Name",
+  Value: "Node Cloud demo"
+};
+
+const ec2 = ncAWS.EC2(options);
```

Comment:
Here it should be `ncAWS.compute`. Reason is what we are providing is a common interface.
So Each provider should implement their own compute and implement `createInstance`, `listInstances` and other relevant calls. Those calls may invoke AWS EC2 create instance as you have done.